numpy/add_newdocs.py

```diff
@@ -668,9 +668,8 @@ def luf(lamdaexpr, *args, **kwargs):
         will be in C-contiguous order (last-index varies the fastest).
         If order is 'F', then the returned array will be in
         Fortran-contiguous order (first-index varies the fastest).
-        If order is 'A' (default), then the returned array may be
-        in any order (either C-, Fortran-contiguous, or even discontiguous),
-        unless a copy is required, in which case it will be C-contiguous.
+        If order is 'A' (default), then the order of the returned array is
```

Comment:
I'm afraid I don't completely understand.

If order is 'A':
if input is Fortran-ordered, output is Fortran-ordered,
if input is C-ordered, output is C-ordered

If order is 'K':
If a copy is made: same memory layout
If not: then same as A (which means Fortran is Fortran and C is C)

So the place where they only differ is non-contiguous arrays?